Client/src/bkr/client/wizard.py

```diff
@@ -744,7 +744,7 @@ def getSkeleton(self): return getNode(self.skeleton)
     def getLicenseContent(self, license):
         content = findNode(self.licenses, "license", license)
         if content:
-            return re.sub("\n\s+$", "", content.firstChild.nodeValue)
+            return re.sub("\n\\s+$", "", content.firstChild.nodeValue)
```

Comment:
Any particular reason why you decided not to use `r` here? 
This regex will remove all trailing spaces including the leading newline.

Reply:
```
>>> LICENSE = """
... TEXT
... TEXT
... TEXT
...
... """
>>>
>>> re.sub("\n\\s+$", "", LICENSE)
'\nTEXT\nTEXT\nTEXT'
>>> re.sub(r"\n\s+$", "", LICENSE)
'\nTEXT\nTEXT\nTEXT'
```

Reply:
my tired mind was scared because of the move of a literal `\n` to a regex-ex `\n` 😂

Reply:
updated!